src/components/Notifications.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Notifications } from '../types/Notifications';
+
+interface Props {
+  notification: Notifications,
+  resetNotification: () => void,
+}
+
+export const Notification: React.FC<Props> = ({
```

Comment:
I think it will be better to name this component Error or ErrorNotification, it will be more understandable